etc/frontend/routes.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
     <router id="standard">
-        <route frontName="yotposmsbump" id="yotposmsbump">
+        <route frontName="yotpo" id="yotposmsbump">
```

Comment:
We are packaging different sub modules under Yotpo namespace. Is it fine using a general term "yotpo" for a component inside the sub module ? Asking this because tomorrow if we need to add a new controller under yotpo-core module, we can't use frontname="yotpo" there. This should be unique between modules. So at that time we might be forced to use the word "yotpocore" there.

Reply:
Thanks for the input.
Was looking at the [default router](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/routing.html) for future enhancement and figured out it should be possible to migrate the router to the core, and the Cart to the core as well.
Used `yotpo_messaging` for now to avoid it.